plugins/rich-editor/src/scripts/editor/EditorContent.tsx

```diff
@@ -49,6 +49,29 @@ export default function EditorContent(props: IProps) {
     return <div className="richEditor-textWrap" ref={quillMountRef} />;
 }
 
+let hljs: any;
```

Comment:
I doubt about variable name. Usually we keep long explicit version of names like: `hightLightJs`